tools/depends/configure.ac

```diff
@@ -421,6 +421,7 @@ esac
 case $use_platform in
   raspberry-pi)
      target_platform=raspberry-pi
+     use_platform_name=rbpi
```

Comment:
please either reuse target_platform instead of inventing yet another confusing var or drop completely, since it's not really needed.